catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -201,6 +202,7 @@ private void initKerberosIfNecessary(Map<String, String> conf, Configuration had
             new ScheduledThreadPoolExecutor(
                 1, getThreadFactory(String.format("Kerberos-check-%s", info.id())));
 
+        KerberosName.resetDefaultRealm();
```

Comment:
What is the purpose of this line?

Reply:
`KerberosName.resetDefaultRealm();` will reset the Kerberos realm, some tests use realm `EXAMPLE.COM` and some use `HADOOPKRB`, in the same JVM,  there can't be more than one realm here or failures in test will occur.